contracts/HydraDelegation/modules/DelegationPoolLib/DelegationPoolLib.sol

```diff
@@ -138,48 +150,15 @@ library DelegationPoolLib {
         return totalRewardsEarned(pool, account) - pool.claimedRewards[account];
     }
 
-    function getRPSValues(
-        DelegationPool storage pool,
-        uint256 startEpoch,
-        uint256 endEpoch
-    ) internal view returns (RPS[] memory) {
-        if (startEpoch > endEpoch) {
-            revert DelegateRequirement({src: "DelegPoolLib", msg: "INVALID_ARGUMENTS"});
-        }
-
-        RPS[] memory values = new RPS[](endEpoch - startEpoch + 1);
-        uint256 itemIndex = 0;
-        for (uint256 i = startEpoch; i <= endEpoch; i++) {
-            if (pool.historyRPS[i].value != 0) {
-                values[itemIndex] = (pool.historyRPS[i]);
-            }
-
-            itemIndex++;
-        }
-
-        return values;
-    }
-
     /**
-     * @notice returns the scaling factor used for decimal places
-     * @dev this means the last 18 places in a number are to the right of the decimal point
-     * @return uint256 the scaling factor
-     */
-    function magnitude() private pure returns (uint256) {
-        return 1e18;
-    }
-
-    /**
-     * @notice returns the amount of rewards earned by an account in a pool
-     * @param rps the reward per share
-     * @param balance the balance of the account
-     * @param correction the correction of the account
-     * @return uint256 the amount of rewards earned by the account
+     * @notice increments the amount rewards claimed by an account
```

Comment:
Most of the notices are starting with a capital letter. Maybe it is a good idea to keep it that way.